### PR TITLE
Added more fields to german Translation (that I missed, sorry!)

### PR DESCRIPTION
**Pull request by fridomac**

### CODE REVIEW

**Comment by bochoven** [comment]
something went wrong, there are merge conflicts: https://github.com/munkireport/munkireport-php/pull/619/files

File: assets/locales/de.json [hunk 1]
```diff
@@ -857,3 +857,391 @@
...
+	"admin": {
+		"bu": {
+			"more_info": "Mehr Informationen"
+		},
+		"bu_overview": "Geschäftsbereiche",
+		"delete_failed": "Löschen fehlgeschlagen: ",
+		"edit_bu": {
+			"title": "Geschäftsbereiche editieren",
+			"name": "Name des Geschäftsbereiches",
+			"address": "Adresse"
```
Comment (bochoven) [inline assets/locales/de.json]: Your missing a comma here

File: assets/locales/de.json [hunk 2]
```diff
@@ -807,7 +807,7 @@
       "soon": "Wartung steht an",
       "nobattery": "Keine Batterie",
       "title": "Batterie-Zustand",
-      "tooltip": "Liste des Batteriezustands"
+      "tooltip": "LIste des Batteriezustands"
```
Comment (bochoven) [inline assets/locales/de.json]: The capital I is back :-)

**Comment by fridomac** [comment]
Hope I did everything right this time, sorry for the hassle....

**Comment by bochoven** [comment]
Excellent!

**Pull request merged**